.github/workflows/JetNews.yaml

```diff
@@ -45,6 +45,10 @@ jobs:
         working-directory: ${{ env.SAMPLE_PATH }}
         run: ./gradlew spotlessCheck assembleDebug lintDebug --stacktrace
 
+      - name: Run local tests
+        working-directory: ${{ env.SAMPLE_PATH }}
+        run: ./gradlew testDebug
```

Comment:
Add path? `path: ${{ env.SAMPLE_PATH }}/app/build/reports/tests`

Reply:
What for? the working directory is set and reports are collected in another task